js/ui/draggable.js

```diff
@@ -858,9 +871,29 @@ var Draggable = DOMComponentWithTemplate.inherit({
         return this["_" + name + "Action"] || this._createActionByOption(name);
     },
 
+    _getAnonymousTemplateName: function() {
+        return "content";
```

Comment:
why not global variable instead of func?

Reply:
This method is virtual and used in base class DOMComponentWithTemplate